include/openthread/nat64.h

```diff
@@ -88,6 +88,121 @@ typedef struct otIp4Cidr
     uint8_t      mLength;
 } otIp4Cidr;
 
+/**
+ * @struct otNat64Counters
+ *
+ * This structure represents the counters for NAT64.
+ *
+ */
+typedef struct otNat64Counters
+{
+    uint64_t m4To6SumPackets;
+    uint64_t m4To6SumBytes;
+    uint64_t m6To4SumPackets;
+    uint64_t m6To4SumBytes;
+} otNat64Counters;
+
+/**
+ * @struct otNat64ProtocolCounters
+ *
+ * This structure represents the counters for the protocols supported by NAT64.
+ *
+ */
+typedef struct otNat64ProtocolCounters
+{
+    otNat64Counters mTotal;
+    otNat64Counters mIcmp;
+    otNat64Counters mUdp;
+    otNat64Counters mTcp;
+} otNat64ProtocolCounters;
+
+/**
+ * Packet drop reasons.
+ *
+ */
+typedef enum otNat64DropReason
+{
+    OT_NAT64_DROP_REASON_ANY = 0,           ///< Packet drop for all reasons.
```

Comment:
The enum value shall be exclusive with each other to avoid confusion? We usually have *_UNKNOWN as a default value. Thus only one value can be chosen for each usage.